code/xcvm/lib/core/src/network.rs

```diff
@@ -14,7 +14,7 @@ use serde::{Deserialize, Serialize};
 pub struct InterpreterOrigin {
 	pub user_origin: UserOrigin,
 	#[serde(with = "hex")]
-	#[schemars(with = "String")]
+	#[cfg_attr(feature = "std", schemars(with = "String"))]
```

Comment:
Does this actually break otherwise? Interesting…

Reply:
if you will return it without attr, it will fail to compile as part of Substrate runtime 100%. But I assume that all which is in core almost certainly wil be used by it. Example? Polkadot -> Compoble -> sudo XCVM  on behalf some origin directly instead of contract. other case sure. Gov precompiles may be. But idea is that at least half of core reusable by substrate/rpc/etc. So it is cross compiles for std and not. 

It will gov way in September as CW will support no_std

Reply:
> It will go away in September as CW will support no_std

Last comment I got a couple days ago is that cosmwasm 2.0 will be ready when it’s ready without them committing to any timeline.  We may need to prepare for continuing to use a fork.